src/driver-gskill.c

```diff
@@ -499,6 +500,9 @@ gskill_get_profile_count(struct ratbag_device *device)
 		return rc;
 	}
 
+	log_debug(device->ratbag,
+		  "Profile count: %d\n", buf[3]);
```

Comment:
this looks like it should fit on one line (same with a few other log_debugs introduced here)